components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/resources/web/siddhitryit/js/sendInputData.js

```diff
@@ -27,15 +27,28 @@ function sendAjaxRequestToSiddhiProcessor(sendInputData) {
                 var jsonObject = JSON.parse(data.responseText);
                 if (jsonObject != undefined) {
                     if (jsonObject.success.localeCompare("true") == 0) {
-                        var jsonArray = JSON.parse(jsonObject.jsonValue);
                         jQuery('.js_resultCol').show();
+                        var jsonArray = JSON.parse(jsonObject.jsonValue);
                         var tabContent = document.getElementById('resultsId');
                         tabContent.innerHTML = '<div id="tabs"><ul id="tabHeaders"></ul></div>';
                         for (i = 0; i < jsonArray.length; i++) {
                             var tabHeader = document.getElementById('tabHeaders');
-                            tabHeader.innerHTML = tabHeader.innerHTML + '<li><a href="#tabs-' + i + '">' + jsonArray[i].key + '</a></li>';
+                            var jsonArrayKey=jsonArray[i].key;
+                            var jsonArrayValue=jsonArray[i].jsonValue;
+                            if(jsonArrayKey.indexOf(':')==0){
```

Comment:
Thanks for pointing out. This has been already fixed in commit https://github.com/thiliA/carbon-event-processing/commit/093f2cedc72bbe0b696dcde85204fa88d262a2bd